### DISCUSSION TITLE
Unable to install on Debian 12 (Bookworm) (Failed to Fetch Archives)

### DISCUSSION THREAD

**Discussion by K1ngw1ng** [cockpit-project/cockpit#20570] [Q&A]
### Explain what happens

1. Run `. /etc/os-release
echo "deb http://deb.debian.org/debian ${VERSION_CODENAME}-backports main" > \
    /etc/apt/sources.list.d/backports.list`
2. Run `apt update`
3. Run `apt install -t ${VERSION_CODENAME}-backports cockpit`
<img width="1255" alt="Screen Shot 2024-06-08 at 1 20 19 PM" src="https://github.com/cockpit-project/cockpit/assets/92333529/24032fcf-4e32-43d5-bbcd-55ea6b58fabb">

### Where is the problem in Cockpit?

Unknown or not applicable

### Server operating system

Debian

### Server operating system version

12

**Comment by martinpitt** [comment]
You apparently already configured backports in /etc/apt/sources.list.d/sources.list, so please follow the "multiple sources" advice and re-drop backports.list again. The missing gnutls package is not at all related to cockpit, this is a broken mirror or missing `apt update` or similar.

**Reply by K1ngw1ng** [reply]
Noted,Thank you for your help. I will reconfigure on my end.
